README.md

```diff
@@ -58,6 +58,15 @@ installers for all supported operating systems.
     sudo apt-get install etcher-electron
     ```
 
+##### Uninstall
+
+```sh
+sudo apt-get remove etcher-electron
+sudo apt-key del D401AB61
```

Comment:
So do you think we should close this PR? Uninstall instructions for these kinds of installations are probably easy to find in the internet, and we can probably handle in case by case basis if really needed.